cg/meta/workflow/microsalt/microsalt.py

```diff
@@ -74,15 +67,15 @@ def get_case_path(self, case_id: str) -> list[Path]:
 
     def get_latest_case_path(self, case_id: str) -> Path | None:
         """Return latest run dir for a microbial case, if no path found it returns None."""
-        lims_project: str = self.get_project(
-            self.status_db.get_case_by_internal_id(internal_id=case_id).links[0].sample.internal_id
-        )
+        case: Case = self.status_db.get_case_by_internal_id(case_id)
+        sample_id: str = case.links[0].sample.internal_id
+        lims_project: str = self.get_project(sample_id)
```

Comment:
Why do we do this workaround case -> sample -> project? Is there no mapping case -> project? This seems to signal that there can be multiple projects for a case, one for each sample, whereupon this mapping could give rise to bugs as well.

Reply:
I could not find a better way to do it atm, leaving it for now